sonarapp/Program.cs

```diff
@@ -16,6 +16,8 @@ static void Main(string[] args)
 
             Boolean whatif = (new verify()).verifywhat("whatif");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'whatif' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS1481)